core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -406,6 +420,42 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
             .pipe(takeUntil(this.destroy$))
             .subscribe((event: DragEvent) => {
                 event.preventDefault(); // Prevent file opening
+
+                const iframeRect = this.iframe.nativeElement.getBoundingClientRect();
+
+                const isInsideIframe =
+                    event.clientX > iframeRect.left && event.clientX < iframeRect.right;
+
+                let direction;
+
+                if (
+                    isInsideIframe &&
+                    event.clientY > iframeRect.top &&
+                    event.clientY < iframeRect.top + 100
+                ) {
+                    direction = 'up';
+                }
+
+                if (
+                    isInsideIframe &&
+                    event.clientY > iframeRect.bottom - 100 &&
+                    event.clientY <= iframeRect.bottom
+                ) {
+                    direction = 'down';
+                }
+
+                if (!direction) {
+                    this.store.updateEditorState(EDITOR_STATE.DRAGGING);
+
+                    return;
+                }
+
+                this.store.updateEditorState(EDITOR_STATE.SCROLLING);
+
+                this.iframe.nativeElement.contentWindow?.postMessage(
+                    { name: 'scroll-inside-iframe', direction },
```

Comment:
create an entry in the `NOTIFY_CUSTOMER` an rename it to `ema-scroll-page` or something like that

Reply:
Added!